segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -169,4 +190,32 @@ private void cleanupDirectories() throws IOException {
     }
 
     //endregion
+
+    public static void main(String[] args) throws Exception {
```

Comment:
I added a comment explaining its purpose. It's so that I can start BK out of process.